src/composite/MegaChart.js

```diff
@@ -46,7 +46,11 @@
     MegaChart.prototype.publish("gutter", 0, "number", "Gap Between Cells",null,{tags:["Basic"]});
     
     MegaChart.prototype.publishProxy("chartType", "_chart", "chartType"); 
-        
+
+    MegaChart.prototype.getChartDataFamily = function () {
+        return this._allCharts[this.chartType()].family;
```

Comment:
This won't work. It would need to be this**._chart**._allCharts[this.chartType()].family

But does **MegaChart** really need **getChartDataFamily**?